Kommunicate-io/Kommunicate-Web-SDK
# FW-3136 Encrypt all the attachment sent from widget

### What do you want to achieve?
- all the media attachments should be encrypted

### PR Checklist


### How was the code tested?

- go to dashboard, send various types of attachments. All of these attachments should be properly displayed in chat widget.
- go to chat widget send different types of attachments. these attachments should be accessible in dashboard.
- repeat same with dialogflow bot

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch
